unit-tests/live/image-quality/test-image-quality.py

```diff
@@ -172,10 +172,20 @@ def is_depth_meaningful(config, laser_enabled=True, save_image=False, show_image
 ################################################################################################
 
 test.start("Testing depth frame - laser ON -", dev.get_info(rs.camera_info.name))
-res, laser_black_pixels = is_depth_meaningful(cfg, laser_enabled=True, save_image=DEBUG_MODE, show_image=DEBUG_MODE)
+res = False
+max_black_pixels = float('inf')
+for i in range(5):
+    result, laser_black_pixels = is_depth_meaningful(cfg, laser_enabled=True, save_image=DEBUG_MODE, show_image=DEBUG_MODE)
+    res = res or result
+    max_black_pixels = min(max_black_pixels,laser_black_pixels)
 test.check(res is True)
 test.finish()
 
-################################################################################################
-
+if res is True:
```

Comment:
Why do we condition it?

Reply:
I figured that if we have no depth in the frame, there is no point in comparing it to the frame without the laser, because the frame might be mostly black pixels anyway

Reply:
And the previous will fail?

Reply:
The comparison? It might fail, yes, not guaranteed but it can happen, but I don't think it indicates anything.
I can remove the condition if needed.

Reply:
Please add a comment explaining it